packages/platform-ios/native_modules.rb

```diff
@@ -5,24 +5,23 @@
 #
 require 'pathname'
 
-def use_native_modules!(root = "..", config = nil)
```

Comment:
This is a breaking change, unless I leave this argument here and print a warning that it does not take any effect and will be removed.

Otherwise, due to the presence of the next argument, `config`, we are potentially running into silent errors.

How about: `if config is string` (means that somebody is passing path to the function), print out a warning and ignore it.

We use `config` parameter for testing purposes only (AFAIK).